test/integration/commands/new-test-slow.js

```diff
@@ -0,0 +1,48 @@
+let test = require('tape')
+let { join } = require('path')
+let tiny = require('tiny-json-http')
+let lib = join(process.cwd(), 'test', 'lib')
+let { begin: _begin, newFolder, run, start, shutdown } = require(lib)
+
+test('Run new tests (slow)', async t => {
+  await run(runTests, t)
+  t.end()
+})
+
+async function runTests (runType, t) {
+  let mode = `[New / ${runType} (slow)]`
+  let begin = _begin[runType].bind({}, t)
+
+  let newAppDir = 'new'
+  let installing = /Installing dependencies/
+  // let appFound = /Existing Begin app already found in this directory/
+  let url, cwd
+
+  t.test(`${mode} new - normal, with live npm install`, async t => {
+    t.plan(3)
+    process.env.__SLOW__ = true
+    cwd = newFolder(newAppDir)
+    let r = await begin('new', cwd)
+    t.notOk(r.stdout, 'Did not print to stdout')
+    t.match(r.stderr, installing, 'Printed dep installation message to stderr')
+    t.equal(r.code, 0, 'Exited 0')
+  })
+
+  t.test(`${mode} Start Begin dev`, async t => {
+    let port = await start[runType](t, cwd)
+    url = `http://localhost:${port}`
+  })
+
+  t.test(`${mode} Check for valid starter project response`, t => {
+    t.plan(1)
+    tiny.get({ url }, function _got (err, result) {
+      if (err) t.fail(err)
+      else t.ok(result.body.includes('https://enhance.dev'), 'Got valid Enhance starter project response')
```

Comment:
@kj just kinda stubbed this in – any other, better means of detecting that the response is indeed being emitted from a functioning default Enhance starter project?

Reply:
Actually, now that I'm thinking about it, maybe running these tests in the Enhance starter project would actually be a better front-line defense?

You'd want Enhance starter project changes to blow up well before they ever got published; if this test failed in the CLI then any bugs likely already slipped through the starter project's tests and likely made it to production.

Fwiw I think we should still keep this test, and others like it, as a backstop here for different reasons (like exercising npm, etc.).